### PR TITLE
Improved export

### PR DESCRIPTION
**Pull request by ianb**
The page looks like this:

image

It's not fancy, but I think it's workable.

### CODE REVIEW

**Comment by 6a68** [comment]
@ianb I thought we were just going to bump the pagination from 10 to 100. Was that decision revisited in a meeting that I missed?

File: locales/en-US/server.ftl [hunk 1]
```diff
@@ -338,6 +338,12 @@ shotDeleteCancel = Cancel
 shotDeleteConfirm = Delete
     .title = Delete
 
+## Export page
+
+exportTitle = Export
+# Note: "File" should match the name of the File Menu, and "Save Page As" should match that menu item. The file will be saved to "exportTitle_files" (it will be "_files" for all locales)
+exportInstructions = To export: use File &gt; Save Page As... and you will find your screenshots in Export_files/
```
[the remark this answers is not in the document]
Comment (ianb) [reply locales/en-US/server.ftl]: OK, fixed in 2c88821aadd7c097246ab3748a58477c469352a5

File: locales/en-US/server.ftl [hunk 2]
```diff
@@ -338,6 +338,11 @@ shotDeleteCancel = Cancel
 shotDeleteConfirm = Delete
     .title = Delete
 
+## Export page
+
+# Note: "File" should match the name of the File Menu, and "Save Page As" should match that menu item. Do not localize "Export_files/"
+exportInstructions = To export: use File > Save Page As... and you will find your screenshots in Export_files/
```
[the remark this answers is not in the document]
Comment (ianb) [reply locales/en-US/server.ftl]: OK, fixed in 2c88821aadd7c097246ab3748a58477c469352a5

File: locales/en-US/server.ftl [hunk 3]
```diff
@@ -338,6 +338,11 @@ shotDeleteCancel = Cancel
 shotDeleteConfirm = Delete
     .title = Delete
 
+## Export page
+
+# Note: "File" should match the name of the File Menu, and "Save Page As" should match that menu item. $folder is replaced with the filename of the folder that will be created
+exportInstructions = To export: use File > Save Page As... and you will find your screenshots in the folder { $folder }
```
Comment (flodolo) [inline locales/en-US/server.ftl]: You still have the 3 dots instead of a single unicode character (`…`).

File: locales/en-US/server.ftl [hunk 4]
```diff
@@ -338,6 +338,11 @@ shotDeleteCancel = Cancel
 shotDeleteConfirm = Delete
     .title = Delete
 
+## Export page
+
+# Note: "File" should match the name of the File Menu, and "Save Page As" should match that menu item. $folder is replaced with the filename of the folder that will be created
```
Comment (flodolo) [inline locales/en-US/server.ftl]: nit: name, not filename

**Pull request merged**